mysql/user_table.sql

```diff
@@ -0,0 +1,20 @@
+-- usersテーブルを作成
+CREATE TABLE users (
+  id INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
+  email VARCHAR(50),
+  password VARCHAR(20),
+  created_at DATETIME,
+  updated_at DATETIME
+  );
+
+-- usersに挿入
+INSERT INTO users
+  (id, email, password, created_at, updated_at)
+values
```

Comment:
Readable/Idiomatic

```sql
INSERT INTO テーブル名 (列1, 列2, 列3, ...)
VALUES (値1, 値2, 値3, ...);
```
上記のように、`INSERT INTO VALUES`で一つのSQLの構文なので、
大文字で統一した方が可読性が高いかな。
```suggestion
VALUES
```